impl_core/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java

```diff
@@ -241,6 +241,9 @@ long getCount() {
 
     private final BucketBoundaries bucketBoundaries;
     private final long[] bucketCounts;
+    // If there's a histogram (i.e bucket boundaries are not empty) in this MutableDistribution,
+    // exemplars will have the same size to bucketCounts; otherwise exemplars are empty.
```

Comment:
Suggestion: It might be clearer to use null when there are no exemplars, since it has a different meaning from zero buckets.  It would also allow null analysis to enforce that the code handles this case.

Reply:
Make sense, updated.